### PR TITLE
Replace buefy snackbars

### PR DESCRIPTION
**Pull request by mradavi**
### Description

Replaced buefy snackbars with custom csc-ui toasts.

### Related issues
GL 944

### Mentions
Part of PR #1006

### CODE REVIEW

**Comment by mradavi** [comment]
Is there a reason the old snackbar had hard-coded strings that are not in lang.js?

`"Encryption engine is ready. Hit refresh to refresh the window to enable encryption.",`

**Comment by csc-felipe** [comment]
> Is there a reason the old snackbar had hard-coded strings that are not in lang.js?

I think it was forgotten. Feel free to extract the strings, or create an issue about it.

**Comment by hannyle** [comment]
I think the PR also needs a rebase

**Comment by mradavi** [comment]
I think now it should be fine. I moved the strings, they'll get translated as part of GL 949.

File: swift_browser_ui_frontend/src/components/ObjectTable.vue [hunk 1]
```diff
@@ -113,6 +113,18 @@
       id="objects-toasts"
       data-testid="objects-toasts"
     />
+    <c-toasts
+      id="largeDownload-toasts"
+      data-testid="largeDownload-toasts"
+    >
+      <p>{{ this.$t("message.largeDownMessage") }}</p>
```
Comment (csc-felipe) [inline swift_browser_ui_frontend/src/components/ObjectTable.vue:120]: As github indicates, no need for `this.` here. `$t` is injected by the plugin, and is available in the templates.

File: swift_browser_ui_frontend/src/components/ObjectTable.vue [hunk 2]
```diff
@@ -113,6 +113,18 @@
...
+      <c-button
+        text
+        @click="removeToast"
+      >
+        {{ this.$t("message.largeDownAction") }}
```
Comment (csc-felipe) [inline swift_browser_ui_frontend/src/components/ObjectTable.vue:125]: same as above

File: swift_browser_ui_frontend/src/components/ObjectTable.vue (hunk 1 above)
Comment (mradavi) [reply swift_browser_ui_frontend/src/components/ObjectTable.vue]: Oops! Thanks for spotting, fixed

File: swift_browser_ui_frontend/src/components/ObjectTable.vue [hunk 3]
```diff
@@ -430,17 +442,19 @@ export default {
...
-      this.$buefy.snackbar.open({
-        duration: 5000,
-        message: this.$t("message.largeDownMessage"),
-        type: "is-success",
-        position: "is-top",
-        actionText: this.$t("message.largeDownAction"),
-        onAction: this.enableDownload,
-      });
+      document.querySelector("#largeDownload-toasts").addToast(
+        { type: "info",
```
Comment (blankdots) [inline swift_browser_ui_frontend/src/components/ObjectTable.vue:452]: the snackbar was success and this became info, is that a design decision or something got lost when moving to it ?

**Review by blankdots — COMMENTED** [review]
just a question

File: swift_browser_ui_frontend/src/components/ObjectTable.vue (hunk 3 above)
Comment (mradavi) [reply swift_browser_ui_frontend/src/components/ObjectTable.vue:452]: For this one with the text phrased as negative
```No large (> 1GiB) downloads enabled. Click to enable them for the duration of the session.```
"success" didn't seem most appropriate. But hopefully we'll get Aino to review all the UI with all the updated elements in the migration :)

Comment (blankdots) [reply swift_browser_ui_frontend/src/components/ObjectTable.vue:452]: maybe warning, but I can live with `info` and Aino making the decision on it.

**Review by blankdots — APPROVED** [review]

**Review by hannyle — APPROVED** [review]

**Pull request merged**